components/ShoppingItem.js

```diff
@@ -75,11 +108,55 @@ const StyledLink = styled(Link)`
   grid-area: details;
   border: 1px solid #362f23;
   border-radius: 15px;
-  height: 1.8rem;
-  width: 4.2rem;
+  height: 1.4rem;
+  width: 3.4rem;
   align-self: center;
   background-color: #fff4e9;
   text-decoration: none;
   color: #362f23;
   padding: 2px 0 0 6px;
+  font-size: 12px;
+`;
+
+const StyledButton = styled.button`
+  grid-area: delete;
+  border: none;
+  background-color: transparent;
+`;
+
+const DeleteIcon = styled(Delete)`
+  width: 20px;
+  height: 20px;
+  border-left: 1px dotted #362f23;
```

Comment:
Oh, this is where the line comes from. I thought it was an accident. So it's a design decision. I would rethink that because it looked to me like it was accidentally cut out not quite right or something. 😉

Reply:
I will implement an edit button in a later story which will also have the dotted line then. Will look better probably when the edit button is added :D